commands/azureCommands/delete-azure-registry.ts

```diff
@@ -19,24 +20,17 @@ export async function deleteAzureRegistry(context?: AzureRegistryNode): Promise<
     } else {
         registry = await quickPickACRRegistry();
     }
+    const shouldDelete = await confirmUserIntent('Are you sure you want to delete this registry and its associated images? Enter yes to continue: ');
+    if (shouldDelete) {
+        let subscription: SubscriptionModels.Subscription = acrTools.getRegistrySubscription(registry);
+        let resourceGroup: string = acrTools.getResourceGroupName(registry);
+        const client = AzureUtilityManager.getInstance().getContainerRegistryManagementClient(subscription);
```

Comment:
Should we make a quick function for this?
getAzureUtilityManager_managementClient(subscription) or something if we're going about simplifying things  + we use this in a ton of places